vars/dockerExecute.groovy

```diff
@@ -133,6 +149,11 @@ void call(Map parameters = [:], body) {
             .mixin(parameters, PARAMETER_KEYS)
             .use()
 
+        config = ConfigurationHelper.newInstance(this, config)
+            .addIfEmpty('dockerSidecarRegistry', config.dockerRegistry)
+            .addIfEmpty('dockerSidecarRegistryCredentials', config.dockerRegistryCredentials)
+            .use()
+
```

Comment:
Just curious: Why not merge the two effective lines between line 149 and 150?

Reply:
Before we can access `config.dockerRegistry` and `config.dockerRegistryCredentials` before having the config. And the config is not available above. In case there is another way to get these properties without having the config, let me know ...

Reply:
Of course, sorry I overlooked that.